### PR TITLE
Ofmcc 1112 - application upload documents

### PR DESCRIPTION
**Pull request by vietle-cgi**
Add upload document to Application's Operating Costs

### CODE REVIEW

File: frontend/src/views/applications/OperatingCostsView.vue [hunk 1]
```diff
@@ -140,10 +164,54 @@ export default {
...
+    },
+
+    async getDocuments() {
+      if (this.model.facilityType != FACILITY_TYPES.RENT_LEASE) return
+      try {
+        this.$emit('process', true)
+        this.processing = true
+        this.uploadedDocuments = await DocumentService.getDocuments(this.$route.params.applicationGuid)
+      } catch (error) {
+        this.setFailureAlert('Failed to get your document', error)
```
Comment (weskubo-cgi) [inline frontend/src/views/applications/OperatingCostsView.vue:179]: Maybe "Failed to retrieve Supporting Documents"?

File: frontend/src/views/applications/OperatingCostsView.vue [hunk 2]
```diff
@@ -140,10 +164,54 @@ export default {
       } catch (error) {
         this.setFailureAlert('Failed to save your application', error)
       } finally {
+        this.processing = false
+        this.$emit('process', false)
+      }
+    },
+
+    async getDocuments() {
+      if (this.model.facilityType != FACILITY_TYPES.RENT_LEASE) return
```
Comment (weskubo-cgi) [inline frontend/src/views/applications/OperatingCostsView.vue:173]: I'd be tempted to add a computed value isRentLease since it's used a fair bit.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/views/applications/OperatingCostsView.vue (hunk 1 above)
Comment (vietle-cgi) [reply frontend/src/views/applications/OperatingCostsView.vue]: it is updated

File: frontend/src/views/applications/OperatingCostsView.vue (hunk 2 above)
Comment (vietle-cgi) [reply frontend/src/views/applications/OperatingCostsView.vue]: it is updated

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**